sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample1_DetectLanguage.md

```diff
@@ -27,10 +27,10 @@ Console.WriteLine($"Detected language {language.Name} with confidence {language.
 
 ## Detecting the language of multiple text inputs
 
-To detect the language of a collection of text inputs in the same language, call `DetectLanguages` on an `IEnumerable` of strings.  The results are returned as a `DetectLanguageResultCollection`.
+To detect the language of a collection of text inputs in the same language, call `DetectLanguagesBatch` on an `IEnumerable` of strings.  The results are returned as a `DetectLanguageResultCollection`.
```

Comment:
Are we staying with DetectLanguageBatch or DetectLanguagesBatch?

Reply:
no `s`. my bad. Will look and fix